frontend/views/containers/modals/DesignSystemBase.vue

```diff
@@ -0,0 +1,98 @@
+<template lang="pug">
+modal-base-template(:class='{ "has-background": background }')
+  .wrapper-container
+    .example-header
+      h2 Modal base example
+
+    .example-container
+
+      form
+        .field
+          i18n.label(tag='label') Full name
+          input.input(value='Felix Kubin')
+
+        .field
+          i18n.label(tag='label') Introduce the potential new member(s) to your group
+          textarea(rows='5')
+            | Felix and Brian are two very important figures in the electronic music scene. They have greatly contributed to the development of genres like ambient music and are now ready to contribute to this group. They are Dreamers like us!'
+
+        .buttons
+          i18n(
+            tag='button'
+            @click.prevent='toggleBackground'
+          ) Toggle background
```

Comment:
@mmbotelho and @pieer when toggling the background here doesn't do anything (because the BG is already grey) but the title alignment isn't the same... shouldn't it be the same?

Reply:
not related

Reply:
Hmmmm... I believe it has something to do with modal-inside-a-modal mechanics... Anyway, it is not working as expected on the Design System but I am assuming that it will on a case-by-case basis, if that makes sense. 

I am not sure if what I'm saying is correct though

Reply:
Oh I though @sandrina-p was talking about text alignment (left or center). Nevermind!